gsuite/config.go

```diff
@@ -6,27 +6,20 @@ import (
 	"log"
 	"runtime"
 
+	"encoding/json"
 	"github.com/hashicorp/terraform/helper/logging"
+	"github.com/hashicorp/terraform/helper/pathorcontents"
 	"github.com/hashicorp/terraform/terraform"
 	"github.com/pkg/errors"
 	"golang.org/x/oauth2/google"
-	directory "google.golang.org/api/admin/directory/v1"
-	"github.com/hashicorp/terraform/helper/pathorcontents"
 	"golang.org/x/oauth2/jwt"
+	directory "google.golang.org/api/admin/directory/v1"
 	"net/http"
 	"strings"
-	"encoding/json"
 )
 
 var defaultOauthScopes = []string{
-	directory.AdminDirectoryCustomerScope,
 	directory.AdminDirectoryGroupScope,
-	directory.AdminDirectoryGroupMemberScope,
-	directory.AdminDirectoryOrgunitScope,
-	directory.AdminDirectoryUserScope,
```

Comment:
Don't need AdminDirectoryUserScope scope for resources like `gsuite_user`?

Reply:
I thought this was tested and not needed, I just verified it myself and yes we do need the UserScope for `gsuite_user`.  Good catch!